lib/helpers.js

```diff
@@ -524,3 +545,7 @@ exports.getBluemixDefaultPlan = function(serviceType) {
     default:                    return 'Lite';
   }
 }
+
+exports.ignoreFile = function(ignoreLine) {
```

Comment:
Some places we use `fs` to read the filesystem, but all the file writing (which is in the refresh generator) should be through `this.fs`.